### ISSUE TITLE
v2.12: oAuth package triggers "old API" warning

### ISSUE THREAD

**Issue by allenfuller** [meteor/meteor#12641]
Running v2.12 with `WARN_WHEN_USING_OLD_API=true` and running into errors from the oAuth package.
```
W20230518-16:20:42.951(-6)? (STDERR)    Calling method meteor_oauth_pendingCredentials.remove from old API on server.
W20230518-16:20:42.951(-6)? (STDERR)    This method will be removed, from the server, in version 3.
W20230518-16:20:42.951(-6)? (STDERR)    Trace is below:
W20230518-16:20:42.952(-6)? (STDERR) [object Object]
W20230518-16:20:42.952(-6)? (STDERR)     at warnUsingOldApi (packages/mongo/collection.js:24:12)
W20230518-16:20:42.952(-6)? (STDERR)     at Collection.remove (packages/mongo/collection.js:752:5)
W20230518-16:20:42.952(-6)? (STDERR)     at _cleanStaleResults (packages/oauth/pending_credentials.js:30:29)
W20230518-16:20:42.952(-6)? (STDERR)     at packages/meteor.js:365:18
W20230518-16:20:42.952(-6)? (STDERR)     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1389:31)
W20230518-16:20:42.952(-6)? (STDERR)     at packages/meteor.js:613:25
W20230518-16:20:42.952(-6)? (STDERR)     at runWithEnvironment (packages/meteor.js:1486:24)
```

Confirmed that [this package](https://github.com/meteor/meteor/blob/devel/packages/oauth/pending_credentials.js) has a `.remove` in it (lines 24-31):
```
// Periodically clear old entries that were never retrieved
const _cleanStaleResults = () => {
  // Remove credentials older than 1 minute
  const timeCutoff = new Date();
  timeCutoff.setMinutes(timeCutoff.getMinutes() - 1);
  OAuth._pendingCredentials.remove({ createdAt: { $lt: timeCutoff } });
};
```

Seems easy enough, I can knock out a PR, just wanted to file an issue first and make sure this is expected.

**Comment by StorytellerCZ** [comment]
@allenfuller yes, please submit a PR.

**Comment by StorytellerCZ** [comment]
Fixed in 2.14
